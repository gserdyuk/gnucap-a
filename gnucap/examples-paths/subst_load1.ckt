* test for substitution
.options dollar_as_spice_comment
.attach $HOME/gnucap/g-plus/g-plus/gnucap-a/gnucap/models-bsim/BSIM3v330/bsim330.so

v1 n1 0 dc 1  
 
.list
.end
