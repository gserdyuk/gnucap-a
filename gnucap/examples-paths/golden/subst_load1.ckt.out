Gnucap gnucap-arails  2009.12.07 RCS 26.136 experimental_1 2010-02-16
The Gnu Circuit Analysis Package
Never trust any version less than 1.0
Copyright 1982-2009, Albert Davis
Gnucap comes with ABSOLUTELY NO WARRANTY
This is free software, and you are welcome
to redistribute it under the terms of 
the GNU General Public License, version 3 or later.
See the file "COPYING" for details.
* test for substitution 
M: already installed, replacing
stashing as M:0
nmos8: already installed, replacing
stashing as nmos8:0
pmos8: already installed, replacing
stashing as pmos8:0
nmos49: already installed, replacing
stashing as nmos49:0
pmos49: already installed, replacing
stashing as pmos49:0
v1 ( n1 0 )  DC  1.
